src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx

```diff
@@ -12,35 +12,30 @@ type InterfaceDrawerDetailsTabProps = {
 };
 
 const InterfaceDrawerDetailsTab: FC<InterfaceDrawerDetailsTabProps> = ({ selectedInterface }) => {
-  const ports = selectedInterface.bridge?.port?.map((port) => port.name);
+  const { t } = useNMStateTranslation();
+  const ports = selectedInterface.bridge?.port?.map((port) => port.name)?.sort();
 
   return (
     <div>
-      {selectedInterface.ethtool?.feature && (
-        <div className="pf-u-mb-md">
-          <Title headingLevel="h4">{t('Features')}</Title>
-          <List isPlain>
-            {Object.keys(selectedInterface.ethtool?.feature)
-              .sort((a, b) => (a > b ? 1 : -1))
-              ?.map((feature) => (
-                <ListItem key={feature}>
-                  <Flex>
-                    <FlexItem>
-                      <Checkbox
-                        isDisabled
-                        id={`checkbox-${feature}`}
-                        value={feature}
-                        isChecked={selectedInterface.ethtool?.feature?.[feature]}
-                      />
-                    </FlexItem>
+      {selectedInterface.vlan && (
+        <div className="pf-u-mb-md" data-test="lldp-section">
+          <Title headingLevel="h4">VLAN details</Title>
 
-                    <FlexItem>{feature}</FlexItem>
-                  </Flex>
-                </ListItem>
-              ))}
-          </List>
+          <Flex>
+            <FlexItem>
+              <strong>{t('Id')}:</strong>
+            </FlexItem>
+            <FlexItem>{selectedInterface.vlan.id}</FlexItem>
+          </Flex>
+          <Flex>
+            <FlexItem>
+              <strong>{t('Base interface')}:</strong>
+            </FlexItem>
+            <FlexItem>{selectedInterface.vlan['base-iface']}</FlexItem>
+          </Flex>
         </div>
       )}
+
       <div className="pf-u-mb-md" data-test="lldp-section">
         <Title headingLevel="h4">LLDP</Title>
```

Comment:
I can see here there's no t function used for LLDP. Don't we need it? I am not sure here...